test/cloudinary-uploader.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { shallow } from 'enzyme'
-import { cloudinaryUploader } from '../src/'
+import cloudinaryUploader, { FORBIDDEN_PATTERN } from '../src/cloudinaryUploader'
```

Comment:
We should define this separately here or just use relevant filename examples to avoid coupling the implementation too closely.

Reply:
My thought process here was that the implementation was tightly coupled to a 3rd party, so decoupling it wouldn't really provide us an advantage? I'm not married to it though if you feel strongly about it